Neos.Http.Factories/composer.json

```diff
@@ -9,7 +9,7 @@
     "require": {
         "php": "^7.1",
         "psr/http-factory": "^1.0",
-        "guzzlehttp/psr7": "^1.4.1, !=1.8.0"
```

Comment:
Why did we _disallow_ 1.8.0?

Reply:
Because of https://github.com/neos/flow-development-collection/pull/2433